website/selectel.erb

```diff
@@ -19,6 +19,9 @@
             <li<%= sidebar_current("docs-selectel-datasource-dbaas-datastore-type-v1") %>>
               <a href="/docs/providers/selectel/d/dbaas_datastore_type_v1.html">selectel_dbaas_datastore_type_v1</a>
             </li>
+            <li<%= sidebar_current("docs-selectel-datasource-dbaas-available-extension-v1") %>>
+              <a href="/docs/providers/selectel/d/dbaas_available_extension_v1.html">selectel_dbaas_available_extension_v1</a>
```

Comment:
These links don't match  the file names in `website/doc/d` directory.
Please rename these files to:
dbaas_datastore_types _v1.html.markdown -> dbaas_datastore_type_v1.html.markdown
dbaas_available_extensions_v1.html.markdown -> dbaas_available_extension_v1.html.markdown 
(both have `s` ending now)